pybamm/solvers/solution.py

```diff
@@ -15,7 +15,6 @@
 class NumpyEncoder(json.JSONEncoder):
     """
     Numpy serialiser helper class that converts numpy arrays to a list
```

Comment:
Since the link is removed, add explanation about why this is required

Reply:
Done